src/components/timeline/ActiveTabTimeline.css

```diff
@@ -28,9 +28,78 @@
 }
 
 .timelineResourcesHeader::before {
-  position: absolute;
   top: calc(50% - 5px);
   left: 8px;
+}
+
+.timelineResourceTracks {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+/* Active tab timeline resource tracks */
+.timelineTrackResource:first-child {
+  border-top: none;
+}
+
+.timelineTrackResourceRow {
+  position: relative;
+  flex-direction: column;
+}
+
+.timelineTrackResourceRow .timelineTrackThreadMarkers {
+  width: 100%;
+}
+
+.timelineTrackResourceRow .threadActivityGraph,
+.timelineTrackResourceRow .timelineMarkers {
+  transition: height 0.2s;
+}
+
+.timelineTrackResourceRow:not(.opened) .timelineMarkers {
+  height: 3px;
+}
+
+.timelineTrackResourceRow.opened
+  .timelineMarkers:not(.timelineMarkersGeckoMain)
+  canvas {
+  height: 6px;
+}
+
+.timelineTrackResourceRow .timelineMarkersGeckoMain {
+  height: 9px;
+}
+
+.timelineTrackResourceRow:not(.opened) .threadActivityGraph {
+  height: 6px;
+}
+
+.timelineTrackResourceLabel {
+  z-index: 3;
```

Comment:
Is this z-index needed? I took it out and everything seemed to not be needed. In addition, generally when we've used a z-index in the past, we've documented why it's needed, and reference the stacking context that it is getting around. z-indexes are also only incremented by 1 as needed in order to clearly show the relationship being expressed. This is mainly since [stacking contexts are kind of ridiculous](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) in how they work.